README.md

```diff
@@ -13,6 +13,8 @@ Metabolic Atlas is a web platform integrating open-source genome scale metabolic
 ## Contributing
 If you would like to contribute to the project, for example if you have ideas for new features, discovered a bug or if you would like to improve the code base, please have a look at [CONTRIBUTING.md](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/CONTRIBUTING.md). All ideas and contributions are highly appreciated.
 
+If you discover any potential vulnerabilities associated with the project, please reach out to us at [contact@metabolicatlas.org](mailto:contact@metabolicatlas.org)
```

Comment:
```suggestion
If you discover any potential vulnerabilities associated with the project, please reach out to us at [contact@metabolicatlas.org](mailto:contact@metabolicatlas.org).
```

Reply:
I think it would be better to use `reach out for us` according to https://idioms.thefreedictionary.com/reach+out+for+us

Reply:
Thank you @MalinAhlberg !

@nanjiangshu I looked at the page you linked to and I think what we are using is correct based on their examples.

"If you would like to request a refund, please reach out to customer service."

vs.

"I reached out for my glass of water, ..."